examples/selection.rs

```diff
@@ -63,7 +64,9 @@ fn main() {
             }
         })
         .expect("Failed to create a window !");
-    window.set_title("Selection".to_string());
+    window.set_title(
+        "Selection example, press c/C p/P to copy/paste from selection/primary clipboard respectively".to_string(),
+    );
```

Comment:
That's a pretty long title, not sure if it'll always be readable. I'd rather print it to stdout, given the rest of the interactions of this example are already on stdout anyway.